src/booking/BookingCalendar.tsx

```diff
@@ -51,7 +52,18 @@ export const BookingCalendar: FunctionComponent<BookingCalendarProps> = ({
       {Array.from(data.entries()).map(([month, dateLocationsPairsForMonth]) => (
         <CalendarSectionContainer key={month}>
           <div className="MonthSection">
-            <h2>{t("calendar.month", { monthString: month })}</h2>
+            <h2>
+              {
+                parse(
+                  "1 " + month,
```

Comment:
@jimmymasaru can you explain what happens where with the `"1 " + month`?

Reply:
I think it's because @jimmymasaru has `d MMMM yyyy` as the date parsing format.